### ISSUE TITLE
"attempt to get argmax of an empty sequence" while comparing model Extreme Greadient Boosting 

### ISSUE THREAD

**Issue by UjwalTalele** [pycaret/pycaret#163]
I am repeatedly getting above error while training a classifier, because of this all next models are not able to train and compare accuracy. Please suggest some hack to exclude this model or a generalised solution using pycaret.
![Annotation 2020-05-21 100157](https://user-images.githubusercontent.com/52368122/82523570-3e1a0800-9b4a-11ea-9873-fff8ab4ed777.jpg)

**Comment by pycaret** [comment]
You can pass blacklist parameter within compare_models. 

Can you share the dataset you are using so I can try to reproduce the error on my end.

**Comment by UjwalTalele** [comment]
Thanks for the response:
here is the link for training dataset, the target variable is "Loan_Status"
https://datahack.analyticsvidhya.com/contest/practice-problem-loan-prediction-iii/download/train-file

**Comment by pycaret** [comment]
@UjwalTalele Can you please upload file here. I have to create an account on AV to get file.

**Comment by UjwalTalele** [comment]
Oh, okay apparently forum dosent support csv files, adding in xlxc format
[train - Copy.xlsx](https://github.com/pycaret/pycaret/files/4663319/train.-.Copy.xlsx)

**Comment by pycaret** [comment]
@UjwalTalele Can you please update pycaret to `2.1.2` and xgboost to `1.1.1` using pip install --upgrade. I was able to run `compare_models` on this dataset successfully.

![image](https://user-images.githubusercontent.com/58118658/92416759-0650c380-f12d-11ea-99d1-eaebd2b59e5c.png)
